docs/libraries/azure-resource-manager/rules/put-operation-response-codes.md

```diff
@@ -0,0 +1,31 @@
+---
+title: put-operation-response-codes
+---
+
+```text title=- Full name-
+@azure-tools/typespec-azure-resource-manager/put-operation-response-codes
+```
+
+Put operations should use the `ArmResourceCreateOrUpdateAsync`, `ArmResourceCreateOrReplaceAsync` or `ArmResourceCreateOrReplaceSync` template. They must have 200, 201, default and no other responses.
```

Comment:
@markcowl I'm not sure what the difference is between `ArmResourceCreateOrUpdateAsync` and `ArmResourceCreateOrReplaceAsync`. They seem to result in the same Swagger.